llmproxy/tests/unit_tests/models/test_llama2.py

```diff
@@ -0,0 +1,50 @@
+import os
+from llmproxy.models.llama2 import Llama2, Llama2Model
+from dotenv import load_dotenv
+
+load_dotenv()
+default_api_key = "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"
+
+
+default_prompt ="What's 1+1?"
+default_system_prompt = "Answer correctly"
+default_model = Llama2Model.LLAMA_2_7B.value
+def test_llama2_constructor() -> None:
+    llama2_Con = Llama2().get_completion
+    assert llama2_Con.message == "No prompt detected"
+
+def test_llama2_prompt() -> None:
+    llama2_emp_promt = ""
+    output = Llama2(prompt=llama2_emp_promt,system_prompt=default_system_prompt,model=default_model)
+    output.get_completion()
+    assert output.message == "No prompt detected"
+
+def test_llama2_api_key() -> None:
+    api_key="LMAO-key"
+    output = Llama2(prompt=default_prompt,system_prompt=default_system_prompt,api_key=api_key,model=default_model)
+    output.get_completion()
+    assert output.message == "Authorization header is correct, but the token seems invalid"
+    
+    
+
+def test_llama2_model() -> None:
```

Comment:
This test won't work, along with all of the other tests. You need to run them and ensure they pass. If they are failing you need to refactor the code or the test accordingly. 

- These tests won't work because you are storing the model in the outputs, not the response of get_completion()